pkg/erda-configs/i18n/autotest.json

```diff
@@ -6,9 +6,10 @@
     "wb.content.autotest.scene.boolean": "随机一个布尔类型 true",
     "wb.content.autotest.scene.upper": "随机一个大写字符串 ABC",
     "wb.content.autotest.scene.lower": "随机一个小写字符串 abc",
-    "wb.content.autotest.scene.mobile": "随机一个电话号码 10000000000",
-    "wb.content.autotest.scene.digital_letters": "随机一个字符串 abc",
-    "wb.content.autotest.scene.letters": "随机一个字符串 abc",
+    "wb.content.autotest.scene.mobile": "随机一个电话号码 18888888888",
+    "wb.content.autotest.scene.digital_letters": "随机一个字符串 Abc123",
+    "wb.content.autotest.scene.letters": "随机一个字符串 Abc",
+    "wb.content.autotest.scene.character": "随机一个字符 a",
```

Comment:
where is this i18n text used?

Reply:
![image](https://user-images.githubusercontent.com/28723047/125266404-6198a700-e338-11eb-86fd-3e36749f8fec.png)

autotest scene mock select